runtime/assembly/modules/storage.ts

```diff
@@ -16,19 +16,23 @@ export namespace Storage {
         const key64: u64 = Serialiser.serialise_result(Utils.toU8Array(u8Key));
         const result: u64 = ext_storage_get_version_1(key64);
         const ptr_size: i32[] = Serialiser.separate_pointer_size(result);
-        return Serialiser.deserialise_input(ptr_size[0], ptr_size[1]);
+        let valueu8: u8[] = Serialiser.deserialise_input(ptr_size[0], ptr_size[1]);
+        if (valueu8[0] == 1){
+            return valueu8.slice(2, valueu8.length);
```

Comment:
Why are you slicing from the second index? I think that we should return everything after the first index (f.e if the value is -> `[1, 12, 98, 97, 114]` where 1 means that it is not empty and 12 is the length of the scale array. I think that we should return `[12, 98, 97, 114]` otherwise we won't be able to decode the result into SCALE String right?

Reply:
Because the Rust side expects the value not as scale-encoded, so there's no need for the size